### DISCUSSION TITLE
is Esp3d and Alexa possible ?

### DISCUSSION THREAD

**Discussion by Grex3D** [luc-github/ESP3D#574] [Questions]
Hello guys, do you know if it is possible to link Alexa with the esp3d ?

I want to use Alexa to receive a notification when my print is done (for exemple).

Thx

**Comment by luc-github** [comment]
currently not I was requesting feedback on new notifications, https://github.com/luc-github/ESP3D/discussions/549

I do not know alexa sdk so cannot say if will be possible

**Reply by Grex3D** [reply]
Thanks Luc for your reply and your work :)

**Reply by luc-github** [reply]
So I need to do some research about alexa API, process and implementation which can take a while
